testsuite/MDAnalysisTests/analysis/test_hydrogenbonds_analysis.py

```diff
@@ -246,11 +271,97 @@ def test_logging_step_not_1(self, universe, caplog):
         caplog.set_level(logging.WARNING)
         hbonds.lifetime(tau_max=2, intermittency=1)
 
-        warning = \
-            "Autocorrelation: Hydrogen bonds were computed with step > 1."
+        warning = ("Autocorrelation: Hydrogen bonds were computed with "
+                   "step > 1.")
         assert any(warning in rec.getMessage() for rec in caplog.records)
 
 
+class TestHydrogenBondAnalysisNoRes(TestHydrogenBondAnalysisIdeal):
```

Comment:
Thanks for doing this, avoiding duplicate code as much as possible really helps us avoid future maintenance debt.